google/cloud/forseti/common/gcp_api/servicemanagement.py

```diff
@@ -121,6 +122,7 @@ def get_formatted_service_name(service_name):
         return service_name
 
 
+@tracing.traced()
```

Comment:
This would create a separate span since it won't be aware of the parent `trace_id`. The tracer should be passed when instanciating this class in `forseti.services.inventory.base.gcp.py` (line 1186).

Reply:
Hmmm, how do I get the tracer in `gcp.py` so that I can pass into this class?